minilib/src/main/scala/java/lang/Throwable.scala

```diff
@@ -0,0 +1,19 @@
+package java.lang
+
+class Throwable(s: String, e: Throwable)
```

Comment:
I assume here the reason is because we do not want the stack trace resolution machinery?

Reply:
Yes, that's right. Especially since it is reachable by the very constructor, because the constructor calls `initStackTrace()`.